R/calcCO2Prices.R

```diff
@@ -14,17 +11,15 @@ calcCO2Prices <- function() {
 
   getNames(x) <- NULL
 
-  # read data used for weight
+  # Read data used for weight
   ceds <- calcOutput("Emissions", datasource = "CEDS2024", aggregate = FALSE)
   ceds <- ceds[, , "Emi|CO2|Energy and Industrial Processes (Mt CO2/yr)"]
-  # for years in the future, use last year available from CEDS
+  # For years in the future, use last year available from CEDS
   ceds <- ceds[, pmin(getYears(x), max(getYears(ceds))), ]
   getYears(ceds) <- getYears(x)
 
-  return(list(
```

Comment:
We agreed as a team to not use that rule and switched off that linter. Therefore, I don't think it makes sense to go through legacy code to change it.